sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -32,6 +32,8 @@
   private Map<String, String> tags;
   private Map<String, Object> extra;
   private Map<String, Object> unknown;
+  private Map<String, String> modules;
```

Comment:
@bruno-garcia is it a `Map` or should we reuse the clas `Package`?

Reply:
Protocol says it's just a map of string name and string version but package at this point coincidentally matches that shape. I'd say let's merge this and discuss that offline